Source/WebCore/platform/network/FormData.cpp

```diff
@@ -322,7 +322,7 @@ static void appendBlobResolved(BlobRegistryImpl* blobRegistry, FormData& formDat
     for (const auto& blobItem : blobData->items()) {
         if (blobItem.type() == BlobDataItem::Type::Data) {
             ASSERT(blobItem.data());
-            formData.appendData(blobItem.data()->data() + static_cast<int>(blobItem.offset()), static_cast<int>(blobItem.length()));
+            formData.appendData(blobItem.data()->bytes().subspan(blobItem.offset(), blobItem.length()));
```

Comment:
Should `appendData` just take a std::span?

Reply:
It does take a span and I am passing it a span.